dimes/dimes.py

```diff
@@ -15,9 +15,9 @@ def __init__(
         self,
         data_values: list,
         name: str = "",
-        color: str | None = None,
+        color: Union[str, None] = None,
         is_visible: bool = True,
-        line_type: str | None = None,
+        line_type: Union[str, None] = None,
```

Comment:
Did you use Union because '|' is not supported in previous versions of Python?

Reply:
@nealkruis

Reply:
Correct.